static/styles/portico/landing_page.css

```diff
@@ -4147,12 +4147,13 @@ nav {
                 width: 18px;
                 height: 18px;
                 margin-right: 5px;
-                margin-top: 2px;
+                margin-top: 4px;
                 background-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="16px" height="16px" viewBox="0 0 24 24"><path d="M12 0c-6.627 0-12 5.373-12 12s5.373 12 12 12 12-5.373 12-12-5.373-12-12-12zm-1.25 17.292l-4.5-4.364 1.857-1.858 2.643 2.506 5.643-5.784 1.857 1.857-7.5 7.643z"/></svg>');
             }
 
             .list-content {
                 margin-left: 22px;
+                line-height: 26px;
```

Comment:
[`line-height` should be unitless](https://allthingssmitty.com/2017/01/30/nope-nope-nope-line-height-is-unitless/).